src/coreComponents/managers/TimeHistory/PackCollection.hpp

```diff
@@ -96,6 +103,8 @@ class PackCollection : public HistoryCollection
   string m_fieldName;
   /// The names of the sets to collect history info from
   string_array m_setNames;
+  /// Override the size of the sets to be collected from
+  localIndex m_setSizeOverride;
```

Comment:
```suggestion
  localIndex m_minimumSetSize;
```
Is this a more appropriate name?